presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -108,6 +108,21 @@ The properties that apply to Hive connector security are listed in the
 :doc:`/connector/hive-security` section for a more detailed discussion of the
 security options in the Hive connector.
 
+HDFS Permissions
+^^^^^^^^^^^^^^^^
+Before running any ``CREATE TABLE`` or ``CREATE TABLE ... AS`` statements
+for Hive tables in Presto, you need to check that the operating system user
+running the Presto server has access to the Hive warehouse directory on HDFS. The Hive warehouse
+directory is specified by the configuration variable ``hive.metastore.warehouse.dir``
+in ``hive-site.xml``, and the default value is ``/user/hive/warehouse``. If that
+is not the case, either add the following to ``jvm.config`` on all of the nodes:
+``-DHADOOP_USER_NAME=USER``, where ``USER`` is an operating system user that has proper
```

Comment:
I understand documenting things as they are, but is there a way to make this a normal Hive config property?

Reply:
@electrum yes. that is fine. I will drop that change from this PR, and come back with a code fix.